bool rr_won
float64 time
float64 plan_time
float64 shortcut_time
